src/gui/sdlmain.cpp

```diff
@@ -4306,10 +4306,10 @@ static void config_add_sdl()
 	        "Set the transparency of the DOSBox Staging screen (0 by default).\n"
 	        "From 0 (no transparency) to 90 (high transparency).");
 
-	pstring = sdl_sec->Add_path("max_resolution", deprecated, "");
+	pstring = sdl_sec->Add_string("max_resolution", deprecated, "");
```

Comment:
This and `viewport_resolution` appear to be unused.  I would expect `deprecated` to do something.  Otherwise why not just remove it?  Are we missing a call to `SetDeprecatedWithAlternateValue` here?

The replacement config value `viewport` also has this incorrect path type by the way:

https://github.com/dosbox-staging/dosbox-staging/blob/3157a8c7f7604430e7e97b8715cbb22a0c84ff9f/src/gui/render.cpp#L1064

Reply:
Try `config -h max_resolution`, or put `max_resolution` in your config, and you'll see why they need to stay.

We want to inform people about deprecated settings in their configs and give helpful tips on how to migrate to the new settings.

Reply:
Also changing the type of `viewport` to string, good pickup 🎉